### PR TITLE
Update lock files after clearing cache

### PR DESCRIPTION
**Pull request by gaya3-zipstack**
## What

- Fix wrong versions of dependent llama-index libs

## Why

- lock files were having outdated versions of llama-index adapter lib even though llama-index lib was updated to 0.10.38

## How

- Clear pdm cache and re-generate lock file

## Can this PR break any existing features. If yes, please list possible items. If no, please explain why. (PS: Admins do not merge the PR without this section filled)

- No

## Database Migrations

- NA

## Env Config

- NA

## Relevant Docs

-

## Related Issues or PRs

-

## Dependencies Versions

-

## Notes on Testing

- Did  pdm run backend and could not notice any import error as mentioned in https://zipstack.slack.com/archives/C06SNT0FTU5/p1718620413856559?thread_ts=1718619074.231729&cid=C06SNT0FTU5

## Checklist

I have read and understood the [Contribution Guidelines]().

### CODE REVIEW

**Comment by Deepak-Kesavan** [comment]
@gaya3-zipstack Looks like you have used the old branch which was merged earlier to main. It is creating merge conflict with main and showing old diff which was already in main.

File: prompt-service/src/unstract/prompt_service/main.py [hunk 1]
```diff
@@ -749,10 +750,12 @@ def _retrieve_context(output, doc_id, vector_index, answer) -> str:
     nodes = retriever.retrieve(answer)
     text = ""
     for node in nodes:
-        if node.score > 0.6:
+        # ToDo: May have to fine-tune this value for node score or keep it
+        # configurable at the adapter level
+        if node.score > 0:
```
Comment (hari-kuriakose) [inline prompt-service/src/unstract/prompt_service/main.py:755]: @gaya3-zipstack Wouldn't recommend to do this by default.

If the current state of an adapter still warrants this change, then can we selectively do this for absolutely required adapters only, and keep the same 0.6 score threshold as default for others?

**Review by hari-kuriakose — COMMENTED** [review]
@gaya3-zipstack LGTM overall.

However, please check the comment.

File: prompt-service/src/unstract/prompt_service/main.py (hunk 1 above)
Comment (chandrasekharan-zipstack) [reply prompt-service/src/unstract/prompt_service/main.py:755]: Another suggestion might be to keep this configurable - either by env or by the user as a challenger setting if we are unsure of what the right one should be

File: prompt-service/pyproject.toml [hunk 2]
```diff
@@ -16,7 +16,6 @@ dependencies = [
     "llama-index==0.10.38",
     "python-dotenv==1.0.0",
     "unstract-sdk~=0.32.0",
-    "unstract-adapters~=0.19.0",
```
Comment (gaya3-zipstack) [inline prompt-service/pyproject.toml:19]: Got added by mistake. Removing it....

**Review by chandrasekharan-zipstack — APPROVED** [review]

**Review by hari-kuriakose — APPROVED** [review]
@gaya3-zipstack Looks good overall.

**Pull request merged**